### DISCUSSION TITLE
Stuck on "loading"

### DISCUSSION THREAD

**Discussion by chad-albrecht** [Blackymas/NSPanel_HA_Blueprint#566] [General]
Running 3.2.1 TFT & Blueprint on esphome 3.2.  Have not been able to get my NSPanel working yet, but feel like I'm close.  NSPanel boots through the start screen, finds HA and pulls the Blueprint version.  Then gets to the "loading..." screen and stops.  ESPhome is running and it seems to be pulling the blueprint down as I can connect the switches to the relays.  Any thoughts?
![IMG_1036](https://user-images.githubusercontent.com/25292337/228676969-149f53ca-3ddf-49f9-ba26-051f767f10ed.jpg)
![IMG_1037](https://user-images.githubusercontent.com/25292337/228676975-b76f5230-b2ec-42a8-b6c3-76dbb5759a8c.jpg)

**Comment by chad-albrecht** [comment]
Also, log from the NSPanel itself (esphome page)
<html>
<body>


17:05:12 | [D] | [api:102] | Accepted 192.168.1.114
-- | -- | -- | --
17:05:12 | [D] | [api.connection:920] | Home Assistant 2023.3.6 (192.168.1.114): Connected successfully
17:05:13 | [D] | [number:054] | 'nspanel Display Brightness' - Setting number value
17:05:13 | [D] | [number:113] | New number value: 100.000000
17:05:13 | [D] | [number:012] | 'nspanel Display Brightness': Sending state 100.000000
17:05:13 | [D] | [number:054] | 'nspanel Display Brightness Dimdown' - Setting number value
17:05:13 | [D] | [number:054] | 'nspanel Display Brightness Dimdown' - Setting number value
17:05:13 | [D] | [number:113] | New number value: 10.000000
17:05:13 | [D] | [number:012] | 'nspanel Display Brightness Dimdown': Sending state 10.000000
17:05:14 | [D] | [nspanel:961] | no page-jump
17:05:14 | [D] | [switch:056] | 'nspanel Nextion inited': Sending state ON
17:05:15 | [D] | [switch:017] | 'nspanel Relay 1 Local Fallback' Turning OFF.
17:05:15 | [D] | [switch:017] | 'nspanel Relay 2 Local Fallback' Turning OFF.
17:05:15 | [D] | [text_sensor:067] | 'nspanel Notification Label': Sending state ''
17:05:15 | [D] | [text_sensor:067] | 'nspanel Notification Text': Sending state ''
17:05:15 | [D] | [text_sensor:067] | 'nspanel Notification Text': Sending state ''
17:05:15 | [D] | [switch:017] | 'nspanel Notification unread' Turning OFF.
17:05:17 | [D] | [text_sensor:067] | 'nspanel Last Click': Sending state 'homepage'
17:05:17 | [D] | [nspanel:957] | start page-timer delay 10
17:05:17 | [D] | [text_sensor:067] | 'nspanel currentpage': Sending state 'home'
17:05:17 | [D] | [text_sensor:067] | 'nspanel currentpage': Sending state 'home'
17:05:17 | [D] | [script:077] | Script 'page_timer' restarting (mode: restart)
17:05:17 | [D] | [nspanel:957] | start page-timer delay 10
17:05:18 | [D] | [text_sensor:067] | 'nspanel Settings Entity': Sending state 'unknown'
17:05:27 | [D] | [nspanel:961] | no page-jump


</body>
</html>

**Reply by scoudibou** [reply]
It happened to me with an unavailble entity status that I've add to the configuration. Are you sure of the status of your entities you put in the blueprint ?

**Reply by chad-albrecht** [reply]
You were on the right track...I did not have a weather device configured.  Once I added OpenWeatherMap, the main screen showed up.
